setup.cfg

```diff
@@ -1,2 +1,6 @@
 [wheel]
 universal = 1
+
+[tool:pytest]
+DJANGO_SETTINGS_MODULE = testapp.settings
+norecursedirs = .git .tox .eggs .cache htmlcov venv*
```

Comment:
The default is `'.*', 'build', 'dist', 'CVS', '_darcs', '{arch}', '*.egg'`, so yeah, mostly. I'll get rid of that.